packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/zoomlevel/CameraZoom.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.zoomlevel;
+
+import android.graphics.Rect;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.math.MathUtils;
+
+public final class CameraZoom {
```

Comment:
Javadoc for the class itself and its public members.

Reply:
Done. 

I have renamed this class to `ZoomUtils` and made it package private. Also I simplified the class moving all state into the `ZoomLevelFeature` class where it belongs. 

This reduces method chains and makes the whole design a lot simpler.